mllib/src/main/scala/org/apache/spark/mllib/clustering/BisectingKMeans.scala

```diff
@@ -212,7 +212,9 @@ class BisectingKMeans private (
       }
       level += 1
     }
-    if(indices != null) indices.unpersist()
+    if (preIndices != null) preIndices.unpersist(blocking = false)
```

Comment:
Do we usually write "blocking ="? there's only one argument.
I'd use braces and put the body on the next line too.